portality/static/widget/fixed_query_src_edges.js

```diff
@@ -0,0 +1,68 @@
+jQuery(document).ready(function($) {
+
+    $.extend(true, doaj, {
+
+    publicSearch : {
+        activeEdges : {},
+
+        buildQuery: function(params){
+            let query;
+            if (params.rawQuery){
+                query = es.newQuery({raw: params.queryString});
+            }
+            else {
+                query = es.newQuery(params.initialParams);
+                query.setQueryString(es.newQueryString(params.queryString));
+                if(params.hasOwnProperty(`sortFilter`)) {
+                    query.setSortBy(es.newSort(params.sortFilter));
+                }
+                if(params.hasOwnProperty(`es_type`)) {
+                    query.addMust(es.newTermFilter({field: "es_type", value: params.es_type}));
+                }
+            }
+            return query;
+        },
+
+        init : function(params) {
+            if (!params) { params = {} }
+
+            let current_domain = document.location.host;
+            let current_scheme = window.location.protocol;
+
+            let selector = params.selector || ".facetview";
+            let search_url = current_scheme + "//" + current_domain + doaj.publicSearchConfig.publicSearchPath;
+
+
+            let components = [
+
+                // results display
+                edges.newResultsDisplay({
+                    id: "results",
+                    category: "results",
+                    renderer : doaj.renderers.newPublicSearchResultRenderer()
+                })
+            ];
+            let e = edges.newEdge({
+                selector: selector,
+                template: doaj.templates.newPublicSearch({
+                    titleBar: false,
+                    resultsOnly: true
+                }),
+                search_url: search_url,
+                manageUrl : true,
```

Comment:
For the widget we should set `manageUrl: false` otherwise it will write stuff into the host system's URL bar